doxygen/Doxyfile.in

```diff
@@ -562,8 +562,7 @@ WARN_LOGFILE           = doxygen.log
 # directories like "/usr/src/myproject". Separate the files or directories
 # with spaces.
 
-INPUT                  = @PROJECT_SOURCE_DIR@/include \
-                         @PROJECT_SOURCE_DIR@/src
+INPUT                  = @DOXYTAG_INPUT@
```

Comment:
This should be `@DOXYGEN_INPUT@` I think

Reply:
Good catch !